hw3/pos_tagger.py

```diff
@@ -39,9 +38,6 @@ def load_data(sentence_file, tag_file=None):
         word = re.sub(r'^\d+,', '', word_line)[1:-1]
         tag = re.sub(r'^\d+,', '', tag_line)[1:-1] if tag_file else None
         if word == '-DOCSTART-':
-            curr_words.append('-DOCSTART-')
```

Comment:
I think we can keep this inserting of DOCSTART in our sentence as I read somewhere on piazza that all tags that appear on train and dev file should be used and when we train it, our POSTagger should predict DOCSTART to have DOCSTART tag. But yeah, a sentence always starts with DOCSTART so we can drop it and add a condition when predicting.